core/includes/update.inc

```diff
@@ -418,6 +418,46 @@ function update_variable_del($name) {
     ->execute();
 }
 
+/**
+ * Uninstalls a module from the database.
+ *
+ * Use this to remove contrib modules that have been merged into Backdrop core.
+ * Be sure to add an entry for that module in backdrop_merged_modules().
+ *
+ * @param string $module
+ *   The name of the contrib module to uninstall.
+ *
+ * @return
+ *   TRUE if the module has been found and uninstalled; otherwise FALSE.
+ */
+function update_uninstall_merged_module($module) {
```

Comment:
@klonos you could make `$module` an `$modules` array here and foreach over it.

Reply:
Thanks @serundeputy 👍 ...I imagine also rename the function to `update_uninstall_merged_modules()` (use plural for "modules")?

Reply:
...my only concern is that as it is currently, I return TRUE if a module has been found and uninstalled, and FALSE if not. What should happen in the case of multiple modules, where some were found, while others not?

Reply:
...should we perhaps leave it as is (single module instead of an array), and then use foreach when we call this function? As in:

```php
foreach ($modules as $module) {
  update_uninstall_merged_module($module);
}
```

Reply:
that works

Reply:
It's pretty unusual we merge more than one module into core at a time (hasn't happened yet), so IMO just handling the single-module use-case seems fine.